src/ccnl-core/src/ccnl-prefix.c

```diff
@@ -79,6 +86,11 @@ ccnl_prefix_free(struct ccnl_prefix_s *p)
     ccnl_free(p);
 }
 
+/**
+ * duplicate a prefix
+ * @param prefix the original prefix
+ * @return the duplicated prefix
```

Comment:
should also mention the error case (", NULL otherwise" or "NULL on error" or detailed "returns NULL when the allocation of memory fails, ...")